modules/mapbox/src/deck-utils.ts

```diff
@@ -83,7 +83,7 @@ export function getDeckInstance({
   }
 
   (deckInstance.userData as UserData).mapboxLayers = new Set();
-  (deckInstance.userData as UserData).mapboxVersion = getMapboxVersion(map);
+  // (deckInstance.userData as UserData).mapboxVersion = getMapboxVersion(map);
```

Comment:
Can you add a warning if we detect version <= 1.2.0?

Reply:
We are unable to detect version <=1.2.0 because they do not expose `version`.

Reply:
The gist of this bug is that when `map.version` is undefined, it is either mapbox <= 1.2.0 or maplibre >= 2.0.0. Before this PR we assume it is mapbox. After this PR we assume it is maplibre.